docs/content/gameobject/reading-and-writing-component-data.md

```diff
@@ -99,12 +102,16 @@ public class WriteHealthBehaviour : MonoBehaviour
 
 1. Declare a field of type `Health` Writer and decorate it with a `[Require]` attribute. 
 
-2. Register a callback for `Reader.ComponentUpdated(ISpatialComponentUpdate update) +=` or for `Reader.<component property name>Updated() +=` during `OnEnable()`.
+2. Register a callback for `Reader.ComponentUpdated +=` or for `Reader.<component property name>Updated +=` during `OnEnable()`.
     *  `Reader.ComponentUpdated` is invoked when any component property is updated.
     *  `Reader.<component property name>Updated` is invoked when a specific component property is updated.
 
 **Note:** 
-`Reader.ComponentUpdated` callbacks are invoked before specific property update callbacks. Callbacks can be deregistered using `Reader.ComponentUpdated(ISpatialComponentUpdate update) -=` and `Reader.<component property name>Updated() -=`. Callbacks are also automatically deregistered when a Reader or Writer is removed. Do not deregister callbacks during `OnDisable()` as that’s an invalid operation.
+`Reader.ComponentUpdated` callbacks are invoked before specific property update callbacks. Callbacks can be deregistered using `Reader.ComponentUpdated -=` and `Reader.<component property name>Updated -=`. Callbacks are also automatically deregistered when a Reader or Writer is removed. Do not deregister callbacks during `OnDisable()` as that’s an invalid operation.
+
+**Known issue warning:** 
+- At the moment, the `ISpatialComponentUpdate update` argument of `Reader.ComponentUpdated` may indicate that a component property was changed even if it was not truly changed. This is a bug and not intended behaviour and will be fixed in an upcoming update.
+- At the moment, `Reader.<component property name>Updated` is may be invoked even if `<component property name>` did not truly change. This is a bug and not intended behaviour and will be fixed in an upcoming update.
```

Comment:
For same reasons as above, suggest:
So, suggest...
"The GDK may invoke `Reader.<component property name>Updated`, even if the `<component property name>` did not change.  This is unintended behavior which we will fix in an upcoming update."